### ISSUE TITLE
Color isn't displayed by `git log` when oh-my-zsh is used with WSL(Ubuntu)

### ISSUE THREAD

**Issue by henniss** [robbyrussell/oh-my-zsh#7760]
`git log` correctly displays colorful output in :

* WSL+bash
* WSL+vanilla zsh
* cygwin+zsh+oh-my-zsh
* zsh+oh-my-zsh on a real linux system.

However, under Windows Subsystem for Linux, when using zsh+oh-my-zsh, the ansi color sequences from `git log` aren't recognized. Piping this output through cat does display colors correctly. `git show` and `git diff` both fail to display colors correctly, but `git status` works just fine, as do all other non-git commands I have tried.

![image](https://user-images.githubusercontent.com/17056237/55975692-5d541f80-5c3f-11e9-808c-c12fb95d7964.png)

Behavior is identical when using WSL TTY, and the default windows terminal. 

This seems to be independent of theme. I encountered this with `robbyrussel` and `ys`, but tried about half a dozen random themes and observed this with every one. 

TERM is set to `xterm-256color`. Ubuntu 18.04, Windows 10, zsh 5.4.2, git 2.17.1

**Comment by mcornella** [comment]
Look at your [`git-config`](https://git-scm.com/docs/git-config) for a pager setting and try disabling that. It might be `core.pager` or `pager.log`, `pager.show`, etc.

**Comment by henniss** [comment]
Ah, that was exactly it. I had configured `LESS=-XF` in my `$ZSH_CUSTOM` dir, when I should have set `LESS=-XFR`, and because of the `-F` option, I didn't realize `less` was even being invoked. 

I should have wiped out `$ZSH_CUSTOM` as soon as I encountered this divergence, but I hadn't made this mistake on other machines and the fact that this was occurring now, with WSL, was a red-herring that I focused on. 

Thanks very much for the help.
